### ISSUE TITLE
Using Command+Shift+G key in Mac using Robotjs

### ISSUE THREAD

**Issue by deepajonnala** [octalmage/robotjs#120]
Hi Octalmage,

I am pretty new to robots and have been working on Automating the upload file functionality lately.
i was following the API's provided and could pretty much understand the Key functions and could implement the same.
But one thing where i am blocked and could not understand is how do we do the keyboard operation "Cmd+shift+g" sequentially using robots????

I tried below things but none of them work.Looks like i am missing something.Could you please help
Try 1:-
robot.keyToggle('command','down');
			robot.keyToggle('shift','down');
			robot.keyToggle('g','down');
				robot.keyToggle('command','up');
			robot.keyToggle('shift','down');
			robot.keyToggle('g','down');

Try2:-
robot.keyToggle('command','down');
robot.keyToggle("g","down","shift"); 

tried different ways around keyToggle and ketTap but i couldn't make it work.
Note:-I am on Mac and when i am on finder i am trying to hit cmd+shift+g which is go to folder.
i.e holding down cmd and shift and pressing g

Command and shift keys are always the modifiers in the key toggle and never go together(atlas in the API's).Please help.

Thanks,
Deep

**Comment by deepajonnala** [comment]
Can some one help me in this :(

**Comment by noonat** [comment]
I don't think this is possible right now. I think you need both modifiers set for the g key press, and the code for passing modifiers only currently allows you to pass one.

**Comment by octalmage** [comment]
**TLDR**
I don't think it's possible until we implement multiple modifiers for keyTap and keyToggle. Sorry! 

**Longer version**

Mac is really weird about key events. For most apps you can't just send CMD, then SHIFT then G. You need to send a G keypress with CMD and SHIFT as modifiers. It's very silly. 

![](https://cldup.com/ExqScP7axC.gif)

As you can see, CMD+SHIFT+G are all pressed, but as you noticed it still doesn't work.

**Comment by noonat** [comment]
@octalmage Would you be open to a pull that changed things to allow multiple modifiers to be passed, perhaps by allowing an array of strings to be passed?

**Comment by noonat** [comment]
Opened #130 with a change to allow multiple modifiers.

**Comment by deepajonnala** [comment]
@octalmage and @nonat Thanks to both of you for looking into this and providing a workaround.Will try with this change today and will let you guys know in case of issues? Thank you !!!

**Comment by octalmage** [comment]
This one was all @nonat! But it would be awesome if you could try it out and let us know how it works! I'll do the same.

**Comment by octalmage** [comment]
This has been resolved by #130, and published to npm in RobotJS v0.3.6. 

Thanks @noonat!
